### ISSUE TITLE
Unable to build and run from master (Maven)

### ISSUE THREAD

**Issue by belu** [gluonhq/scenebuilder#348]
With the latest master I am unable to build and start SceneBuilder. Last time I tried with Gradle, this worked out-of-the-box.

### Expected Behavior
All dependencies should be resolved and SceneBuilder should start.

### Current Behavior
The build fails with the following error:
`[ERROR] Failed to execute goal on project app: Could not resolve dependencies for project com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT: Failed to collect dependencies at com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Failed to read artifact descriptor for com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Could not find artifact com.gluonhq.scenebuilder:parent:pom:16.0.0-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]`

### Steps to Reproduce
Checkout latest master. Run the following command:
`mvn javafx:run -f app`

### Your Environment
Linux

### Log
```
$ mvn javafx:run -f app
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] --------------------< com.gluonhq.scenebuilder:app >--------------------
[INFO] Building Scene Builder App 16.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> javafx-maven-plugin:0.0.5:run (default-cli) > process-classes @ app >>>
Downloading from gluon-nexus: https://nexus.gluonhq.com/nexus/content/repositories/releases/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/maven-metadata.xml
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/maven-metadata.xml
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/parent-16.0.0-SNAPSHOT.pom
Downloading from gluon-nexus: https://nexus.gluonhq.com/nexus/content/repositories/releases/com/gluonhq/scenebuilder/parent/16.0.0-SNAPSHOT/parent-16.0.0-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.071 s
[INFO] Finished at: 2021-04-05T16:04:33+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project app: Could not resolve dependencies for project com.gluonhq.scenebuilder:app:jar:16.0.0-SNAPSHOT: Failed to collect dependencies at com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Failed to read artifact descriptor for com.gluonhq.scenebuilder:kit:jar:16.0.0-SNAPSHOT: Could not find artifact com.gluonhq.scenebuilder:parent:pom:16.0.0-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by Oliver-Loeffler** [comment]
Hi @belu!

I just tried that with a fresh Ubuntu 20.04 install and Maven 3.8.1 and I share your experience completely.
Started to play around with configuring my repositories etc. and all failed. Then I decided to drop everything and start overwith all Maven defaults.

Starting conditions:
* Maven 3.8.1
* Java 16
* deleted `~/.m2/repositories`
* deleted `~/.m2/settings.xml`

Sequence of commands:

```bash
$ cd workspace
$ git clone https://github.com/gluonhq/scenebuilder.git
$ cd scenebuilder
$ mvn install
$ mvn javafx:run -f app
```

This worked out. The important point was to run `mvn install` really at first. Then, everything works fine. This was repeatable.
There should be no need, to run `mvn clean install -f kit`. If only this command is executed before attempting the app to run, dependency resolution will fail. If I run the kit build first, then I am again facing the issue as described by you.

Following sequence fails:
```bash
$ mvn clean install -f kit
$ mvn javafx:run -f app
```

Also this one fails:
```bash
$ mvn clean package
$ mvn javafx:run -f app
```

This one works:
```bash
$ mvn install
$ mvn javafx:run -f app
```
